shell/exercism.fish

```diff
@@ -0,0 +1,49 @@
+# Configure
+complete -f -c exercism -n "__fish_use_subcommand" -a "configure" -d "Writes config values to a JSON file."
+complete -f -c exercism -n "__fish_seen_subcommand_from configure" -s t -l token -d "Set token"
+complete -f -c exercism -n "__fish_seen_subcommand_from configure" -s w -l workspace -d "Set workspace"
+complete -f -c exercism -n "__fish_seen_subcommand_from configure" -s a -l api -d "set API base url"
+complete -f -c exercism -n "__fish_seen_subcommand_from configure" -s s -l show -d "show settings"
+
+# Download
+complete -f -c exercism -n "__fish_use_subcommand" -a "download" -d "Downloads and saves a specified submission into the local system"
+complete -f -c exercism -n "__fish_seen_subcommand_from download" -s e -l exercise -d "the exercise slug"
+complete -f -c exercism -n "__fish_seen_subcommand_from download" -s h -l help -d "help for download"
```

Comment:
We can probably do without tab completion for the help flag, as it will repeat the information you have already added for each option. Thoughts?

Reply:
these completions fire when a user types `exercism <command> -<tab>` (note the dash). It's only there for completeness, I created these mostly by copying the output of `exercism help <command>`.